spec/features/basic_search_spec.rb

```diff
@@ -17,15 +25,15 @@
         given(:keyword) {'monster'}
 
         scenario "Title contains inputted keyword Upcase" do
-          expect(subject.first_result_title).to include(keyword.capitalize)
+          expect(@page.search_results.first.title.text).to include(keyword.capitalize)
```

Comment:
Create a separate method to get the title like `@page.first_result_title`. Focus to not manage elements directly, only by SitePrism page object accessors.